doc/contribute/external.rst

```diff
@@ -164,3 +170,32 @@ The flowchart below shows an overview of the process:
    :align: center
 
    Submission process
+
+.. _external-tooling:
+
+Contributing External Tooling
+*****************************
+
+This section deals exclusively with the inclusion of external tooling in the
+Zephyr project, where tooling is defined as software that assists the
+compilation, testing or simulation processes but in no case ends up being part
+of the code compiled and linked into the final image.
+
+There are no license requirements for tooling components, but licenses that
+have not been approved by the `Open Source Initiative (OSI)`_ might be rejected.
+
+Just like with regular external components, tooling that is imported from
+another project can be integrated either in the main tree or as a :ref:`west
+project <west-workspace>` Note that in this case the corresponding west project
+will not be a :ref:`module <modules>`, because tooling does not make use of the
+Zephyr build system nor needs to be processed by it.
+
+If the tool is integrated in the main tree it should be placed under the
+:file:`scripts/` folder.
+If the tool is integrated as a west project, then the project repository can be
+hosted outside the zephyrproject-rtos GitHub organization, provided that the
+project is made optional via the `group-filter:` field in the main `west.yml`
+manifest. More info on optional projects can be found in :ref:`this section
+<west-manifest-groups>`.
+
+The TSC must approve every Pull Request that introduces a new external tool.
```

Comment:
I would add a note about importing manifests. Changes to the imported manifests, i.e. adding new projects or repos must be reviewed by the TSC as well. We should not have manifests go beyond what was initially approved.